slm_porosity/heat_transfer.f90

```diff
@@ -826,4 +906,25 @@ FUNCTION Spline_cubic (r_p, l_p, fr_p, fl_p, dfr_p, dfl_p)
       - fr_p*l_p**3 - r_p**2*(3*fl_p*l_p - dfl_p*l_p**2 + dfr_p*l_p**2))/(r_p-l_p)**3
   END FUNCTION Spline_cubic
 
+  FUNCTION three_parameter_connection (temperature, liquid_fraction, der_solid, der_liquid, fusion, is_D)
```

Comment:
What is `connection`? Which scientific paper use this terminology? There is no such word in manual; therefore, use `three_parameter_model`. It is far from the best, but acceptable. It will be fine, if you can suggest your ideas in comments, but not in code.

Reply:
I have already suggested to use `(..., Dsolid, Dliquid, fusion_jump, ...)`. You have agreed by default, because there is no your comments about this question in the discussion. Therefore, do as we agreed. Again. If you have any suggestions, write them in comments, not in code.